lib/powerpoint/slide/text.rb

```diff
@@ -79,7 +79,7 @@ def render_elem
         before = %(<a:r><a:rPr dirty="0" lang="en-US" #{sz} #{bold} smtClean="0">#{color}</a:rPr><a:t>)
         after = '</a:t></a:r>'
 
-        "#{before}#{values.join}#{after}"
+        "#{before}#{values.map { |b| b.encode(:xml => :text) }.join}#{after}"
```

Comment:
I would prefer this pattern be pulled out to a helper method, though it gets tricky since you're using ERb here.

```ruby
module OutputEncoding
  def encode_xml(text)
    text.encode(xml: :text)
  end
end
```

Actually, I think if you include this module into Util (or just put the method itself there) it will be available to the templates as well since the erb runs in the current binding `renderer.result(binding)`